config/db.js

```diff
@@ -8,6 +8,7 @@ module.exports = {
     host: config.host,
     port: config.port,
     dialect: 'postgres',
+    timezone: 'America/Bogota',
```

Comment:
Database should be in UTC o GMT 0, otherwise it will be hard to handle more than one time zone